pkg/config/piped.go

```diff
@@ -466,22 +465,54 @@ type PipedPlatformProvider struct {
 	Name string                     `json:"name"`
 	Type model.PlatformProviderType `json:"type"`
 
-	KubernetesConfig *PlatformProviderKubernetesConfig `json:"kubernetesConfig,omitempty"`
-	TerraformConfig  *PlatformProviderTerraformConfig  `json:"terraformConfig,omitempty"`
-	CloudRunConfig   *PlatformProviderCloudRunConfig   `json:"cloudRunConfig,omitempty"`
-	LambdaConfig     *PlatformProviderLambdaConfig     `json:"lambdaConfig,omitempty"`
-	ECSConfig        *PlatformProviderECSConfig        `json:"ecsConfig,omitempty"`
```

Comment:
Why did we remove these tags?

Reply:
It's not being used anyway, since when we marshal the object to JSON, only one of them values is being used and displayed as `config` in the output of MarshalJSON function.
![Screen Shot 2022-08-01 at 18 14 58](https://user-images.githubusercontent.com/32532742/182115670-6fa63af1-37aa-423a-bcd3-d302bc6ff99b.png)

Reply:
Nice point. Got it.